docs/Command-Line-Interface.md

```diff
@@ -70,37 +75,52 @@ Or, if no header is provided, the default hostname and port (set in the env file
   <tr>
     <td>load</td>
     <td>load "path"</td>
-    <td>Loads a file or a directory in TachyonFS into memory.</td>
+    <td>Load a file or a directory in TachyonFS into memory.</td>
+  </tr>
+  <tr>
+    <td>loadMetadata</td>
+    <td>loadMetadata "path"</td>
+    <td>Load a file or a directory metadata from underlying file system into Tachyon.</td>
   </tr>
   <tr>
     <td>location</td>
     <td>location "path"</td>
-    <td>Displays a list of hosts that have the file data.</td>
+    <td>Display a list of hosts that have the file data.</td>
   </tr>
   <tr>
     <td>ls</td>
     <td>ls "path"</td>
-    <td>List all the files and directories directly under the given path with information such as size.</td>
+    <td>List all the files and directories directly under the given path with information such as
+    size.</td>
   </tr>
   <tr>
     <td>lsr</td>
     <td>lsr "path"</td>
-    <td>Recursively list all the files and directories under the given path with information such as size.</td>
+    <td>Recursively list all the files and directories under the given path with information such
+    as size.</td>
   </tr>
   <tr>
     <td>mkdir</td>
     <td>mkdir "path"</td>
-    <td>Create a directory under the given path, along with any necessary parent directories. This command will fail if the given path already exists.</td>
+    <td>Create a directory under the given path, along with any necessary parent directories. This
+    command will fail if the given path already exists.</td>
+  </tr>
+  <tr>
+    <td>mount</td>
+    <td>mount "path" "uri"</td>
+    <td>Mount the underlying file system path "uri" into the Tachyon namespace as "path".</td>
```

Comment:
Then only metadata or real data will be loaded into Tachyon? and if I write data to a file under the mounted "path", will it be transparently written to under storage with the same path?